static/sass/styles.scss

```diff
@@ -4,8 +4,8 @@ $color-brand: #e95420;
 $color-accent: #2c001e;
 
 // global components
-@import '../components/canonical-global-nav/build/css/main';
-@import '../components/canonical-cookie-policy/build/css/cookie-policy';
+@import '../../node_modules/global-nav/build/css/main';
+@import '../../node_modules/cookie-policy/build/css/cookie-policy';
```

Comment:
You don't need `../../node_modules` here as you have `--include-path node_modules` in the sass command.